src/settings_table.cpp

```diff
@@ -116,6 +116,28 @@ static void SettingsValueAbsolute(const IntSettingDesc &sd, uint first_param, in
 	SetDParam(first_param + 1, abs(value));
 }
 
+/** Settings Default Value displays the correct units or as a percentage */
+static void SettingsValueText(const IntSettingDesc &sd, uint first_param, int32_t value)
```

Comment:
This is a really generic name for a quite specific function. Shouldn't it be named something related to servicing interval since it's only used for those settings?

Reply:
Indeed, I'll make it more specific.